specs/derivation.md

```diff
@@ -517,23 +517,30 @@ New frames for timed-out channels are dropped instead of buffered.
 
 The channel-bank can only output data from the first opened channel.
 
-Upon reading, first all timed-out channels are dropped.
+Upon reading, first all expired channels are removed from the known complete channels.
+
+A channel is expired if:
+`current_l1_block.number > channel.open_l1_block.number + max(SEQUENCING_WINDOW_SIZE, CHANNEL_TIMEOUT)`
```

Comment:
min?

Reply:
No we want the max of the two. Min is essentially what the op-node is doing now - treating channels as unknown as soon as they timeout.  We want to extend the period that the channel is known so extra frames that are received for those channels can be ignored, rather than have them start a new channel that we have to wait for it to timeout.

Reply:
Note that the channel will still timeout after `CHANNEL_TIMEOUT` - from then until the end of the sequencing window any frames for that channel will be immediately dropped. After the sequencing window frames with that channel ID will be treated as a new channel again.